.renovaterc.json

```diff
@@ -79,6 +80,16 @@
                 ]
             }
         },
+        {
+            "matchManagers": [
+                "pre-commit"
+            ],
+            "postUpgradeTasks": {
+                "commands": [
+                    "find template -type f -name '.pre-commit-config.yaml.jinja' -exec sed -i '/{{{replace \"/\" \"\\/\" depName}}}/{N;s/\\(rev: \"*{{{currentValue}}}\"*\\)/rev: {{{newValue}}}/}' {} +"
```

Comment:
I would suggest have a simplier regex pattern. What do you think of '/{{{replace \"/\" \"\\/\" depName}}}/{N;s/rev: {{{currentValue}}}/rev: {{{newValue}}}/}'? Just removing the optional quote matching and unnecessary group pattern.

Reply:
@huxuan  I will test it with `{{{replace "/" "\/" depName}}}` later. But replace `s/\\(rev: \"*{{{currentValue}}}\"*\\)` with `s/rev: {{{currentValue}}}` implies version in the field `rev: ` cannot use double quotes.

Reply:
Actually, the changes of replace part is a typo, but I also want to confirm whether the escape is necessary.Feel free to let me know if it does not work.